twake/frontend/src/app/scenes/Error/Error.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Languages from 'services/languages/languages';
+import Emojione from 'components/Emojione/Emojione';
+import LoginService from 'services/login/login.js';
+import './Error.scss';
+import RouterServices from 'app/services/RouterServices';
+
+export default () => {
+  let state = RouterServices.history.location.state as any;
+  if (!state || !state?.error?.name) {
+    state = {
+      error: {
+        name: '404',
+        info: 'Error not found',
+      },
+    };
+  }
+  return (
+    <div className="full_page_error" key="page_error">
+      <div className="error_message skew_in_top_nobounce">
+        <div className="title">
+          <Emojione type=":boom:" /> {Languages.t('scenes.aie', [], 'Aïe !')}
+        </div>
+        <div className="subtitle">
+          {Languages.t(
+            'scenes.error_on_twake',
+            [],
+            'Vous avez trouvé une erreur sur la plateforme Twake !',
```

Comment:
Why not having them in EN by default?

Reply:
Old component from startup, when all of our clients were french.
You can translate it or remove the fallback Stephane.